.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: Build
+on: [push, pull_request]
+jobs:
+  build:
+    strategy:
+      matrix:
+        java: [8-jdk, 11-jdk, 15-jdk]
```

Comment:
Please don't run it three times. Once or twice, for oldest supported (8) and newest supported (16) is OK.